test.sh

```diff
@@ -4,11 +4,11 @@ set -e
 
 cd "$(dirname "$0")"
 
-docker build -t test-runner .
+# docker build -t test-runner .
```

Comment:
Could you revert this change?

```suggestion
docker build -t test-runner .
```

Reply:
Already did 😓

Reply:
Missed by a couple minutes.